xds/src/test/java/io/grpc/xds/internal/security/SecurityProtocolNegotiatorsTest.java

```diff
@@ -178,7 +178,7 @@ protected void onException(Throwable throwable) {
             future.set(throwable);
           }
         });
-    assertThat(executor.runDueTasks()).isEqualTo(1);
+    assertThat(executor.forwardTime(600L, TimeUnit.SECONDS)).isEqualTo(1);
```

Comment:
Why was this changed?

Reply:
With the latest version the behavior is the same, so I just reverted it back (it was needed for 'initial load in constructor' version)